common/src/main/java/org/apache/apex/common/util/ToStringStyle.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.apex.common.util;
+
+import org.apache.hadoop.classification.InterfaceStability.Evolving;
+
+@Evolving
+public abstract class ToStringStyle extends org.apache.commons.lang.builder.ToStringStyle
```

Comment:
It is a base common class for Apex ToStringStyle. In future, if we want to modify common behavior for all ToStringStyle classes, it can be done in this class.